docs/sources/operations/query-acceleration-blooms.md

```diff
@@ -152,7 +152,7 @@ Example calculation for storage requirements of blooms for a single tenant.
 
 Since reading blooms depends heavily on disk IOPS, Bloom Gateways should make use of multiple, 
 locally attached SSD disks (NVMe) to increase i/o throughput. 
-Multiple directories on different disk mounts can be specified using the `-bloom.shipper.working-directory` [setting][gateway-cfg] 
+Multiple directories on different disk mounts can be specified using the `-bloom.shipper.working-directory` [setting]storage-config-cfg] 
```

Comment:
Sorry, I miss the left bracket here, should I raise another PR?

Reply:
Yes, this one is already merged.  Thanks for catching that!